pkg/vm/engine/tae/stl/containers/strvec.go

```diff
@@ -1,104 +1,114 @@
-// Copyright 2022 Matrix Origin
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-// http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
 package containers
 
 import (
 	"fmt"
 	"io"
-	"unsafe"
 
+	"github.com/matrixorigin/matrixone/pkg/container/types"
 	"github.com/matrixorigin/matrixone/pkg/vm/engine/tae/stl"
 )
 
-func NewStrVector[T any](opts ...*Options) *StrVector[T] {
+func NewStrVector2[T any](opts ...*Options) *StrVector[T] {
```

Comment:
Why do you need a 2?

Reply:
Added a new one, then deleted the old one, forgot to change the name back, it has been changed